src/App.js

```diff
@@ -1,8 +1,15 @@
-import { Box, CssBaseline } from '@mui/material';
+import { useState } from 'react';
```

Comment:
Just a general comment, not for this file in particular, but could we set a max width for the logo and title ("Snowprint"). On my laptop it looks a bit oversized. Of course, this is an aesthetic suggestion. Not urgent!